types/validator_address.go

```diff
@@ -28,7 +28,7 @@ type ValidatorAddress struct {
 
 var (
 	// ErrNoHostname is returned when no hostname is set for the validator address
-	ErrNoHostname = errors.New("no hostname")
+	ErrNoHostname error
```

Comment:
Why this change? It means `ErrNoHostname` is nil

Reply:
revert